frontend/src/concepts/connectionTypes/ConnectionTypeForm.tsx

```diff
@@ -31,6 +31,8 @@ const getConnectionTypeSelectOptions = (
   isPreview: boolean,
   selectedConnectionType?: ConnectionTypeConfigMapObj,
   connectionTypes?: ConnectionTypeConfigMapObj[],
+  disableTypeSelection?: boolean,
```

Comment:
nit pick. Imo this isn't an accurate name for doing logic. Because in the preview form we also disable the type selection. A more accurate boolean would say the existing type reference was missing. (Also i could be wrong, but I don't think this boolean is needed at all for this logic?)